Project/DAQ_System/Program/Src/Sensor/Accelerometer/lsm303dlhc.hpp

```diff
@@ -94,6 +94,8 @@ class LSM303DLHC: public IAccelerometer {
 //==============================================================================================
 
     private:
+        //I2C Device Address
+        uint8_t _dev_add = 0x28;
```

Comment:
Make this a constant using `static constexpr`. And rename to something such as `kDeviceAddress`.  [Style Guide](https://google.github.io/styleguide/cppguide.html#Constant_Names).